packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -233,6 +235,11 @@ export const SquigglePlayground: FC<PlaygroundProps> = ({
   const vars = useWatch({
     control,
   });
+
+  useEffect(() => {
+    onSettingsChange?.(vars);
+  }, [vars]);
```

Comment:
onSettingsChange should be in deps too.

Reply:
deps?

Reply:
The `[vars]` array, it should be `[vars, onSettingsChange]`.
Since `useEffect` fires on an initial render (and onSettingsChange gets called), it should get called again if the component's consumer decides to change it.
It's not really important for this specific PR, though.